python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -619,7 +619,11 @@ def assembleMetadata(self, coaddExposure, tempExpRefList, weightList):
             self.inputRecorder.addVisitToCoadd(coaddInputs, tempExp, weight)
         coaddInputs.visits.sort()
         if self.config.doPsfMatch:
-            psf = self.config.modelPsf.apply()
+            # set PSF to modelPsf with maximum dimensions
```

Comment:
In addition to stating what the code is doing here, could you add a brief comment as to the "why"?